wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -189,29 +189,84 @@ Combo# (Integer List) is a list of three numbers, each from 0 - 255 defining an
 Hit Objects
 -----------
 
-Hit Circle Syntax:
+**Hit Circle Syntax:**
 
 `x,y,time,type,hitSound,addition`
+
 `164,260,2434,1,0,0:0:0:0:`
 
-Slider Syntax:
+A hit circle is a single hit in all osu! game modes.
+
+*x (Integer)* ranges from 0 to 512 (inclusive) and *y (Integer)* ranges from 0 to 384 (inclusive).
+
+*time (Integer)* is in milliseconds from the beginning of the song.
+
+*type (Integer)* is a bitmap for the hitobject type:
+
+Bit 0 (1) = circle, bit 1 (2) = slider, bit 2 (4) = new combo, bit 3 (8) = spinner. Bits 4-6 (16, 32, 64) form a 3-bit number (0-7) that chooses how many combo colors to skip. Bit 7 (128) is for an osu!mania hold note. Circles, sliders, and spinners can be OR'd with new combos and the combo skip value, but not with each other.
+
+`1` - circle, `5` - circle starting a new combo, `22` - slider starting a new combo, skipping 2 colors.
+
+*hitSound (Integer)* is a bitmap of hitsounds to play on top of the normal hitsound:
```

Comment:
could be ``hitSounds is a bitmap of hitsounds to play on top of the normal hitsound, in integer value:``